src/hooks/useInlineErrors.test.ts

```diff
@@ -17,11 +17,11 @@ describe("useInlineErrors", () => {
   it("can update an error", () => {
     const { result, rerender } = renderHook(() => useInlineErrors());
     let [errors, setError] = result.current;
-    setError("Error1", "Uh oh!");
+    act(() => setError("Error1", "Uh oh!"));
     rerender();
```

Comment:
Now that these are using `act()` I don't think they need to also have `rerender()`.

Reply:
Removed all the `rerender()` bits.